app/lib/features/comments/models.dart

```diff
@@ -0,0 +1,5 @@
+enum NewCommentLocation {
```

Comment:
Why not have it on actual widget where it's used? OR maybe in ```/utils```. Placing it under ```/models``` sounds a bit confusing...

Reply:
```/models``` in general refers to mapped data objects. Going through ```/models```, one would except some data class.